zoo-css-challenge/style.css

```
@@ -85,45 +90,71 @@ footer {
   border-radius: 50%;
   display: inline-block;
   position: fixed;
-  bottom: 20px;
+  /* bottom: 20px; */
   right: 20px;
-
+  top: 20px;
   display: flex;
   justify-content: center;
   align-content: center;
   flex-direction: column;
+  transition: all 0.25s ease-in;
 }
 
 #badge:hover {
   background-color: yellow;
+  transition: all 0.25s ease-in;
```

```suggestion
  transition: background-color 0.25s ease-in;
```
Instead of doing `all`, it is good practise to choose only the CSS property we want to change. One of the reasons is that performance can be better